### PR TITLE
 add statement for windows users

### PR DESCRIPTION
**Pull request by maicss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,8 +213,9 @@ $ make watch
 $ cd ../myapp
 $ npm run
 ```
+## for Windows user
 
-**Tip**: If you're on Windows and can't use `make`, try [this Gist](https://gist.github.com/mantis/bb5d9f7d492f86e94341816321500934).
+`react-admin` not support Windows for now, [see this](https://github.com/marmelab/react-admin/issues/2030#issuecomment-405909164). Please use [admin-on-rest](https://github.com/marmelab/admin-on-rest) instead.
```
Comment (jdreesen) [inline README.md]:
```diff
- `react-admin` not support Windows for now [...]
+ `react-admin` does not support Windows for now [...]
```

**Comment by fzaninotto** [comment]
No, we do support Windows for users, we just don't support it for developers. I don't agree with your warning.

**Comment by maicss** [comment]
e....most users on github are development, right? So what users you mean? The HTML create by `react-admin` can view on Windows browser?

`we just don't support it for developers`, what's the tips, `make` for Windows means?

**Comment by djhi** [comment]
> most users on github are development, right? So what users you mean?

It means you don't need to build `react-admin` to use it in your own projects on windows. What's currently not supported is working on the `react-admin` project itself on Windows. I'm looking into making it easier though

**Pull request closed without merging**